### ISSUE TITLE
MisconfigurationException for val_loss

### ISSUE THREAD

**Issue by sadmanomee** [EmperorJia/EquiCSP#1]
raise MisconfigurationException(
lightning_fabric.utilities.exceptions.MisconfigurationException: ReduceLROnPlateau conditioned on metric val_loss which is not available. Available metrics are: ['train_loss', 'train_loss_step', 'lattice_loss', 'lattice_loss_step', 'coord_loss', 'coord_loss_step', 'loss_replace_l', 'loss_replace_l_step', 'loss_replace_f', 'loss_replace_f_step', 'loss_score', 'loss_score_step', 'lr-Adam', 'train_loss_epoch', 'lattice_loss_epoch', 'coord_loss_epoch', 'loss_replace_l_epoch', 'loss_replace_f_epoch', 'loss_score_epoch']. Condition can be set using `monitor` key in lr scheduler dict
Epoch 0: 100%|██████████| 106/106 [01:20<00:00,  1.31it/s, v_num=d39l, train_loss_step=2.600, lattice_loss_step=0.721, coord_loss_step=0.892, loss_replace_l_step=0.0688, loss_replace_f_step=0.021, loss_score_step=0.893, train_loss_epoch=3.360, lattice_loss_epoch=0.953, coord_loss_epoch=1.050, loss_replace_l_epoch=0.0886, loss_replace_f_epoch=0.255, loss_score_epoch=1.010]

there is no val_loss in the whole codebase.

**Comment by sadmanomee** [comment]
Solved using "lr_scheduler": {
                "scheduler": scheduler,
                "monitor": "val_checkpoint_on",  # Changed to val_checkpoint_on
                "interval": "epoch",  # Update after each epoch
                "frequency": 1,  # Check every epoch
                "strict": False,  # Allow if metric is missing initially
            },
current version does not work for ReduceLROnPlateau for latest version of pytorch-lightning.
